utility/service/service.go

```diff
@@ -1,11 +1,217 @@
 package service
 
 import (
+	"errors"
+	"fmt"
+
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/runtime/configs"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
 	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/modules/base_modules"
+)
+
+var (
+	errValidateBlockHeight = errors.New("relay failed block height validation")
+	errValidateRelayMeta   = errors.New("relay failed metadata validation")
+
+	_ Servicer = &servicer{}
 )
 
+type Servicer interface {
```

Comment:
NIT: What are your thoughts on moving this into `utility_module.go` so when someone comes to the codebase, they can immediately see what kind of interfaces each actor exposes?